[docs] Refactor "Selectors" (#4799)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/selector/addcustomdomproperties.md

```text
Selector().addCustomDOMProperties({
    property1: fn1,
    property2: fn2,
    /* ... */
}) → Selector
```

Parameter                     | Type     | Description
----------------------------- | -------- | -----------
`property1`, `property2`, ... | String   | Property names.
`fn1`, `fn2`, ...             | Function | Functions that calculate property values. Executed on the client side in the browser.

Functions that calculate property values (`fn1`, `fn2`, ...) take the following parameters.

Parameter   | Type     | Description
----------- | -------- | -----------
`node`      | Object   | The DOM node.

**Example**

```js
import { Selector } from 'testcafe';

fixture `My fixture`
    .page `https://devexpress.github.io/testcafe/example/`;

test('Check Label HTML', async t => {
    const label = Selector('label').addCustomDOMProperties({
        innerHTML: el => el.innerHTML
    });

    await t.expect(label.innerHTML).contains('input type="checkbox" name="remote"');
});
```

If you use TypeScript, declare a Selector interface extension with your custom properties:

<!--

> The `redcarpet` library ignores `ts` code and renders it as a plain text without code highligting.

Reviewer: ```suggestion
The `redcarpet` library ignores `ts` code and renders it as a plain text without highlighting code.
```

> We can't use `js` here too. It's rendered wrong because of the type casting syntax. `csharp` looks

Reviewer: here too?

Reviewer: ```suggestion
You cannot use `js` here either. It is rendered incorrectly because of the type casting syntax. `csharp` looks
```

Author: This comment is not displayed for the users.

> ok for TypeScript code highlighting so we'll use it until we are not fixed the problem with `redcarpet`.

Reviewer: looks ok?

Author: This comment is not displayed for users. It's only visible in markdown.
